src/layouts/EditHomepage.jsx

```diff
@@ -564,82 +596,127 @@ export default class EditHomepage extends Component {
+                                  button={section.resources.button}
+                                  sectionIndex={sectionIndex}
+                                  deleteSection={this.deleteSection}
+                                  onFieldChange={this.onFieldChange}
+                                />
+                                <NewSectionCreator
+                                  sectionIndex={sectionIndex}
+                                  hasResources={hasResources}
+                                  createSection={this.createSection}
+                                />
+                              </div>
+                            )}
+                          </Draggable>
+                        ) : (
+                          null
+                        )}
+
+                        {/* Infobar section */}
+                        {section.infobar ? (
+                          <Draggable
+                            draggableId={`infobar-${sectionIndex}-draggable`}
+                            index={sectionIndex}
+                          >
+                            {(draggableProvided) => (
+                              <div
+                                {...draggableProvided.draggableProps}
+                                {...draggableProvided.dragHandleProps}
+                                ref={draggableProvided.innerRef}
+                              >
+                                <EditorInfobarSection
+                                  title={section.infobar.title}
+                                  subtitle={section.infobar.subtitle}
+                                  description={section.infobar.description}
+                                  button={section.infobar.button}
+                                  url={section.infobar.url}
+                                  sectionIndex={sectionIndex}
+                                  deleteSection={this.deleteSection}
+                                  onFieldChange={this.onFieldChange}
+                                />
+                                <NewSectionCreator
+                                  sectionIndex={sectionIndex}
+                                  hasResources={hasResources}
+                                  createSection={this.createSection}
+                                />
+                              </div>
+                            )}
+                          </Draggable>
+                        ) : (
+                          null
+                        )}
+
+                        {/* Infopic section */}
+                        {/* TO-DO */}
+
+                        {/* Carousel section */}
+                        {/* TO-DO */}
+
+                      </>
+                    ))}
+                    {droppableProvided.droppable}
+                  </div>
+                  /* eslint-enable react/jsx-props-no-spreading */
+                )}
```

Comment:
need to add `{droppableProvided.placeholder}`

Reply:
Resolved with commit d254839